narwhals/_duckdb/dataframe.py

```diff
@@ -159,7 +159,7 @@ def select(
         ):
             return self._from_native_frame(
                 self._native_frame.aggregate(
-                    [val.alias(col) for col, val in new_columns_map.items()]
+                    [val.alias(col) for col, val in new_columns_map.items()]  # type: ignore[arg-type]
```

Comment:
based on the duckdb stubs, only `str` are allowed in `.aggregate`. 🤔

Reply:
looks wrong, it should be `str | list[Expression]`, right? want to make a PR to duckdb?